sentry/src/middleware/cors.rs

```diff
@@ -0,0 +1,164 @@
+use hyper::header::{ACCESS_CONTROL_ALLOW_HEADERS, ACCESS_CONTROL_REQUEST_HEADERS};
+use hyper::{Body, HeaderMap, Method, Request, Response, StatusCode};
+
+pub enum CorsResult {
```

Comment:
This is easier than using `Option<CorsResult>` since we don't need to unwrap the `Option` and add a dummy `Simple` response, we can directly `match` against `CorsResult`!

Reply:
I am not sure it's advisable to go against normal conventions. Having `Option<CorsResult>` makes it easy for anyone new to the codebase to understand

Reply:
Removed.